.github/workflows/linters.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           fetch-depth: 200
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
Should `v20` also be part of the testing matrix?

Reply:
CI consumes a lot of time so it's not feasible rn.

I doubt it has much difference TBH.